aw-server/src/endpoints/mod.rs

```diff
@@ -86,6 +111,18 @@ fn server_info(config: &State<AWConfig>, state: &State<ServerState>) -> Json<Inf
     })
 }
 
+fn get_file(file: PathBuf, state: &State<ServerState>) -> Option<(ContentType, Vec<u8>)> {
+    let asset = state.asset_resolver.resolve(&file.display().to_string())?;
+
+    let content_type = file
+        .extension()
+        .and_then(OsStr::to_str)
+        .and_then(ContentType::from_extension)
```

Comment:
This automatic ContentType detection in theory is only needed for the "static" folder, the rest of them could be hardcoded and use state.asset_resolver.resolve directly. Would make things a little bit easier to read.

Reply:
Here I've just reimplemented `NamedFile` responder functionality.
So I've tried to adjust the code, this is what the routes get instead
```rust
    state
        .asset_resolver
        .resolve(&Path::new("css").join(file))
        .map(|state| (ContentType::CSS, state))
```
which looks alright. Not sure how substantial to regard, but there is a problem with 
- `fonts` folder which has `ContentType::WOFF` and `ContentType::WOFF2` files.
- `js` folder which has `ContentType::JavaScript` and `ContentType::JSON` (`.js.map` files).
- also yes, `static` folder. Have you suggested to move the content detection only to `fn root_static`?

Reply:
Alright, in that case it was probably good as is if detection is pretty much needed for all but the css folder.